sandbox/urls.py

```diff
@@ -36,7 +36,7 @@
     ),
     path(r"sitemap.xml", sitemap, {"sitemaps": {"cmspages": CMSSitemap}}),
     re_path(
-        r"api/{}/".format(API_PREFIX),
+        fr"api/{API_PREFIX:s}/",
```

Comment:
is it a french translation ?

Reply:
no "f" is for "f-string" and "r" for "regex" :man_shrugging:

Reply:
![](https://media.giphy.com/media/l0ErFgOht0LZMHjRm/giphy.gif?cid=ecf05e47diaxo7xb9vb0rjps0us2yhkneyym15i8w00szdfq&rid=giphy.gif&ct=g)

Reply:
<img src="https://user-images.githubusercontent.com/1427165/134137480-27a595fe-8e8a-4968-b98b-31c913e407de.png" width="200"/>